crates/driver/src/domain/competition/solution/settlement.rs

```diff
@@ -215,6 +215,31 @@ impl Settlement {
             .into()
     }
 
+    /// CIP38 score denominated in the native token (ETH)
+    pub fn score_cip38(
```

Comment:
I renamed current score computation to be "old_score" while the cip38 version will be called just "score". 

I did it this way (added another struct) to avoid handling a nightmare of dependencies for the SolverScore enum, and it's soo much easier to do a follow up cleanup because we would just delete the old code (and not modify the existing code and do another iteration of testing), so creating a small temporary mess is justified in this case IMO.